packages/services/f-braze-adapter/package.json

```diff
@@ -35,23 +37,25 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "appboy-web-sdk": "2.5.2",
     "date-fns": "2.16.1",
     "lodash.findindex": "4.6.0",
     "lodash.orderby": "4.6.0",
     "lodash.startswith": "4.2.1",
     "lodash.uniq": "4.5.0"
   },
   "devDependencies": {
+    "@braze/web-sdk": ">=3.3.0",
```

Comment:
Sorry, this one should stay as it was, with no range or caret — only the `peerDependencies` need that.

Reply:
my bad sorted it